js/cards/highlander.txt

```diff
@@ -54,7 +56,9 @@ Nadu, Winged Wisdom 1
 Natural Order 1
 Oath of Druids 1
 Orcish Bowmasters 1
+Phlage, Titan of Fire’s Fury 1
```

Comment:
`Phlage, Titan of Fire's Fury 1`

Reply:
That's what I get for copying card names from somewhere other than scryfall/gatherer